aws/readme.md

```diff
@@ -7,16 +7,66 @@ After completing these steps, please follow the required steps in the [project r
-        * `instance_type`: The instance type which redpanda will be deployed on. Default: `i3.8xlarge`.
-        * `prometheus_instance_type`: The instance type which prometheus and grafana will deployed on. Default: `c5.2xlarge`.
-        * `public_key_path`: Provide the path to the public key of the keypair used to access the nodes. Default: `~/.ssh/id_rsa.pub`
-        * `distro`: Linux distribution to install (this settings affects the below variables). Default: `ubuntu-focal`
-        * `distro_ami`: AWS AMI to use for each available distribution.
-        These have to be changed with according to the chosen AWS region.
-        * `distro_ssh_user`: User used to ssh into the created EC2 instances.
-
-  Example: `terraform apply -var="instance_type=i3.large" -var="nodes=3"`
+  
+Example: `terraform apply -var="instance_type=i3.large" -var="nodes=3"`
+
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| aws | 3.29.1 |
+| local | 2.1.0 |
+| random | 3.1.0 |
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| aws | 3.29.1 |
+| local | 2.1.0 |
+| random | 3.1.0 |
+
+## Modules
+
+No Modules.
+
+## Resources
+
+| Name |
+|------|
+| [aws_instance](https://registry.terraform.io/providers/hashicorp/aws/3.29.1/docs/resources/instance) |
+| [aws_key_pair](https://registry.terraform.io/providers/hashicorp/aws/3.29.1/docs/resources/key_pair) |
+| [aws_security_group](https://registry.terraform.io/providers/hashicorp/aws/3.29.1/docs/resources/security_group) |
+| [local_file](https://registry.terraform.io/providers/hashicorp/local/2.1.0/docs/resources/file) |
+| [random_uuid](https://registry.terraform.io/providers/hashicorp/random/3.1.0/docs/resources/uuid) |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| aws\_region | The AWS region to deploy the infrastructure on | `string` | `"us-west-2"` | no |
+| distro | The default distribution to base the cluster on | `string` | `"ubuntu-focal"` | no |
+| distro\_ami | n/a | `map(string)` | <pre>{<br>  "amazon-linux-2": "ami-01ce4793a2f45922e",<br>  "debian-buster": "ami-0f7939d313699273c",<br>  "debian-stretch": "ami-072ad3956e05c814c",<br>  "fedora-31": "ami-0e82cc6ce8f393d4b",<br>  "fedora-32": "ami-020405ee5d5747724",<br>  "rhel-8": "ami-087c2c50437d0b80d",<br>  "ubuntu-bionic": "ami-0c1ab2d66f996cd4b",<br>  "ubuntu-focal": "ami-02c45ea799467b51b"<br>}</pre> | no |
+| distro\_ssh\_user | The default user used by the AWS AMIs | `map(string)` | <pre>{<br>  "amazon-linux-2": "ec2-user",<br>  "debian-buster": "admin",<br>  "debian-stretch": "admin",<br>  "fedora-31": "fedora",<br>  "fedora-32": "fedora",<br>  "rhel-8": "ec2-user",<br>  "ubuntu-bionic": "ubuntu",<br>  "ubuntu-focal": "ubuntu"<br>}</pre> | no |
+| enable\_monitoring | Setup a prometheus/grafana instance | `bool` | `true` | no |
+| instance\_type | Default redpanda instance type to create | `string` | `"i3.2xlarge"` | no |
```

Comment:
`Instance type used for redpanda nodes`?

Reply:
Yes, the same values that variables.tf